lib/Interpreter/Value.cpp

```diff
@@ -231,9 +231,15 @@ namespace cling {
   namespace valuePrinterInternal {
     std::string printTypeInternal(const Value& V);
     std::string printValueInternal(const Value& V);
+    extern const char* const kUndefined;
   } // end namespace valuePrinterInternal
 
   void Value::print(llvm::raw_ostream& Out, bool Escape) const {
+    if (!m_Interpreter) {
+      Out << valuePrinterInternal::kUndefined << "\n";
```

Comment:
That was changed back in the last commit.
If <<<invalid>>> is the only thing necessary, can the `return "ERROR in cling's callPrintValue(): missing value string.";` be put back to `<undefined>`, trapping 3 cases seems a bit much for no new information...